tools/Importer/readme.md

```diff
@@ -0,0 +1,36 @@
+# Importer
+
+The Importer is a .net console application that loads data to the FHIR server at high speed. It improves the data loading speed significantly through batch multi-thread processing. The tool is very easy to use. Download the code, update the settings in the app.config file, compile and run.
+
+Note that the Importer targets Cosmos DB backend primarily and can accomplish high throughput rates as noted in the app.config file. You will not obtain such high throughput rates with SQL database mainly because resources in bundles are processed one by one, which is not optimized for SQL performance.
+

```

Comment:
Note that though the Importer will work with both gen1 (with Cosmos DB) and gen2 (with SQL DB) APIs, high throughput rates (as noted in the app.config) can currently be obtained with gen1 API only. For gen2 throughput is limited because API processes resources in bundles one by one. Once gen2 API starts using sets in SQL database (planned work) this limitation will be removed.

For now if you consider gen2 with SQL backend, you can use...

Reply:
Revised the doc per comments.